.github/workflows/mkdocs.yml

```diff
@@ -0,0 +1,42 @@
+name: Deploy MkDocs to gh-pages
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Set environment variable
+        id: plum
+        run: echo "::set-output name=PULL_NUMBER::$(jq --raw-output .pull_request.number $GITHUB_EVENT_PATH)"
```

Comment:
@KeisukeShima What is `plum`?

Reply:
I heard from @KeisukeShima that this has no meaning. I'll create another PR to fix this.